freshmaker/pyxis.py

```diff
@@ -0,0 +1,253 @@
+        include_fields = ['data.channel_name', 'data.version',
+                          'data.related_images', 'data.bundle_path_digest']
+        request_params['include'] = ','.join(include_fields)
+        for index_image in index_images:
+            path = index_image.get('path', '')
+            if not path:
+                continue
+            request_params['filter'] = f"source_index_container_path=={path}"
+            bundles = self._pagination('operators/bundles', request_params)
+            bundles_per_index_image[path] = bundles
+
+        return bundles_per_index_image
+
+    def get_bundles_per_channel_per_index_image(self, index_images, params={}):
+        """
+        Get bundle images per channel per index image
+
+        :param list index_images: list of index images
+        :param dict params: parameters to add to GET request
+        :return: bundle images per channel per index image
+        :rtype: dict
+        """
+        bundles_per_index_image = \
+            self._get_bundles_per_index_image(index_images, params)
+
+        ret = {}
+        for index_image, bundles in bundles_per_index_image.items():
+            bundle_per_channel = {}
+            # get latest versions of bundle images per channel
+            for bundle in bundles:
+                channel = bundle.get('channel_name')
+                if channel in bundle_per_channel:
+                    new_ver = bundle.get('version')
+                    old_ver = bundle_per_channel[channel].get('version')
+                    # Check if the right format of version is used
+                    try:
+                        if version.parse(new_ver) > version.parse(old_ver):
+                            bundle_per_channel[channel] = bundle
+                    except version.InvalidVersion as e:
+                        log.warning("Other format than SemVer is used")
+                        log.warning(repr(e))
+                else:
+                    # Check if the right format of version is used
+                    try:
+                        version.parse(bundle.get('version'))
+                        bundle_per_channel[channel] = bundle
+                    except version.InvalidVersion as e:
+                        log.warning("Other format than SemVer is used")
+                        log.warning(repr(e))
+
+            ret[index_image] = bundle_per_channel
+        return ret
+
+    def get_digests_by_nvrs(self, nvrs, params={}):
+        """
+        Get images' digests(manifest_list_digest field) by their nvrs
+
+        :param list nvrs: list of nvrs of ContainerImages to query Pyxis
+        :param dict params: parameters to add to GET request
+        :return: set of digests
+        :rtype: set
+        """
+        include_fields = ['data.brew', 'data.repositories.manifest_list_digest']
+        request_params = copy.deepcopy(params)
+        request_params['include'] = ','.join(include_fields)
```

Comment:
Rather than define `include_fields`, I think you can just do this:
```suggestion
        request_params['include'] = 'data.brew,data.repositories.manifest_list_digest'
```